Detectors/TRD/base/include/TRDBase/Calibrations.h

```diff
@@ -93,6 +94,11 @@ class Calibrations
   bool isChamberNotCalibrated(int det) const { return mChamberStatus->isNotCalibrated(det); }
   char getChamberStatusRaw(int det) const { return mChamberStatus->getStatus(det); }
 
+  //online gain tables.
+  const double getOnlineGainAdcdac(int det, int row, int mcm);
+  const double getOnlineGainFGAN(int det, int row, int mcm, int adc);
+  const double getOnlineGainFGFN(int det, int row, int mcm, int adc);
```

Comment:
const should be placed after the function

Reply:
done

Reply:
fixed and the assocated implementations.